modules/ocf_printhost/files/enforcer

```diff
@@ -122,9 +122,7 @@ NOTIFY_PROBLEMATIC_FILE_SOURCE = dedent("""\
         Please download your file as a PDF and print from desktop instead of web browser.\
 """)
 
-PROBLEMATIC_FILE_SOURCE = [
-    ('Google Docs', lambda job: job.doc_name.endswith(' - Google Docs'))
-]
+PROBLEMATIC_FILE_SOURCE = []
```

Comment:
I think we will still want this. That was a revert of a revert.

Reply:
Yep https://github.com/ocf/puppet/commit/c055f88e1fc7a851f97c6cd4c6fe9cb74fdc22fa